core/src/test/java/com/onelogin/saml2/test/logout/LogoutRequestTest.java

```diff
@@ -735,18 +735,18 @@ public void testIsInValidSign() throws Exception {
 		assertTrue(logoutRequest.isValid());
 
 		settings.setStrict(false);
-		String signature2 = "vfWbbc47PkP3ejx4bjKsRX7lo9Ml1WRoE5J5owF/0mnyKHfSY6XbhO1wwjBV5vWdrUVX+xp6slHyAf4YoAsXFS0qhan6txDiZY4Oec6yE+l10iZbzvie06I4GPak4QrQ4gAyXOSzwCrRmJu4gnpeUxZ6IqKtdrKfAYRAcVf3333=";
+		String signature2 = "x3Yq1dQ0S/6iirAPpkEYrDvY5mTqzQ3b1eE+sEmnmYbzDs5YHksRrc7uloHt7xqBcCGlk+ZI2USjKshf//OVRkSr8gZ8qYtth1v69hVpEvUdzhSANyJCOCENN2DhX8kc76Wg+VyR1mzbvbrap0G6lrj9TSuM4wyh68gzJDeTQbs=";
```

Comment:
Why an update on the dependencies alter the result of a Signature?

Reply:
The old signature2 failed with the validation test (`a valid base 64 alphabet but not a possible value. Expected the discarded bits to be zero.`), that is why I updated it. 

```
testIsInValidSign(com.onelogin.saml2.test.logout.LogoutResponseTest): expected:<[Signature validation failed. Logout Response rejected]> but was:<[Last encoded character (before the paddings if any) is a valid base 64 alphabet but not a possible value. Expected the discarded bits to be zero.]>
```


Maybe the dependency upgrade started to capture it?

Reply:
I verified that the new version of commons-codec 1.15 fixes the issue